action.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Melos action'
```

Comment:
Nit: I think we don't need ' on those values

Reply:
Removed them! (copied the subosito action and didn't think about them being everywhere)